examples/external_request/with_harvester.rb

```diff
@@ -0,0 +1,84 @@
+require 'net/http'
+require 'bundler'
+Bundler.require
+
+require "new_relic/telemetry_sdk"
+
+unless ENV["API_KEY"]
+  raise "No License key supplied.  Export API_KEY environment variable!" 
+end
+
+def setup_buffer_harvesting common_attributes = {test_attribute: 'example'}
```

Comment:
It'd be nice if we could change this to be something more descriptive. Even just `{common_attribute: 'example'}`. We could even use a more realistic example like `{host: 'fake_host'}`. Whatever you think would be clearer to users of this example app if they're running it and sending spans.